.build-test-rules.yml

```
@@ -8,7 +8,7 @@ host/class:
 
 host/class/uvc/usb_host_uvc/examples/camera_display:
   enable:
-    - if: (IDF_VERSION_MAJOR >= 5 and IDF_VERSION_MINOR >= 3)
+    - if: (IDF_VERSION >= "5.3.0")
```

```suggestion
    - if: IDF_VERSION >= 5.3
```

Works locally for me